tests/PHPUnit/System/expected/test_FlattenReports__Referrers.getWebsites_day.xml

```diff
@@ -34,6 +38,8 @@
 		<sum_visit_length>1</sum_visit_length>
 		<bounce_count>0</bounce_count>
 		<nb_visits_converted>0</nb_visits_converted>
+		<Referrers.Website>www.referrer2.com</Referrers.Website>
```

Comment:
Invalid XML

Reply:
actually it's not invalid as `.` is valid for xml tags as long as it doesn't begin with `.`. But I'll replace that with `_`